### ISSUE TITLE
[BUG]: Serious accessibility issues in RSS Guard 5 (Development Build) on Windows 11

### ISSUE THREAD

**Issue by amirsol81** [martinrotter/rssguard#1941]
### Brief description of the issue

I am currently testing the latest RSS Guard 5 development build on Windows 11 25H2, and unfortunately I’ve encountered several serious accessibility issues that make the application very difficult—or impossible—to use for blind and visually impaired users.
Screen readers such as NVDA and JAWS cannot read the article titles in the article tree view, and the text of articles, when opened in the Internal browser, isn't available to screen reader users either.

### Steps to duplicate
Simply go to the article tree, and try navigating via Up/ Down arrows. And try opening articles via the internal browser.

### Expected behavior
Articles should be read upon navigating in the tree view, and the text of articles in various tabs should be presented to screen reader users.

### What happened?
Article names aren't read at all, and we may hear "tree view item 1 of 20" or nothing at ll. Also the article text is completely inaccessible and cannot be viewed or read in the internal browser.

### Logs
Not accessible or applicable here.

### System and app version
OS: Windows 11 Pro 25H2
RSS Guard: 5.0.0-9d49734bb

**Comment by amirsol81** [comment]
@martinrotter I'd sincerely appreciate it if you could take a look at these serious accessibility bugs affecting RSS Guard V5 development builds. There are other relatively minor ones as well, such as not reading items inside menus when menus with submenus are opened via the keyboard, But I've reported the most serious ones here. Many blind users would love to use RSS Guard as it's undoubtedly the best Windows RSS reader available.

**Comment by rkingett** [comment]
These are issues on Windows 10 as well, with the latest version of NVDA.
